src/utils/event.ts

```diff
@@ -88,3 +88,11 @@ export const eventTimeComparator = (a: VenueEvent, b: VenueEvent) => {
 
   return a.duration_minutes - b.duration_minutes;
 };
+
+export const getEventDayRange = (daysInBetween: number) => {
+  if (daysInBetween === 2) {
+    return daysInBetween + 1;
+  }
+
+  return daysInBetween + 2;
+};
```

Comment:
can you explain the meaning of these magic numbers? and give the meaningful names or add a comment describing the logic?

Reply:
yeah, basically `differenceInDays` returns days in between of the target timeline, so we need to add certain amount of days that depends on the day difference result, thus letting us have proper timeline

Reply:
added comments in b41ef61